packagecore.yaml

```diff
@@ -26,15 +26,28 @@ packages:
         - python3-requests
         - python3-urllib3
         - python3
+# centos no beautifulsoup4
```

Comment:
@jarun Yeah it appears to be in default repo. So packagecore installs the generated package after compiling it in order to make sure it works and deps work correctly. Centos can't seem to find the package even after compiling successfully. I need to look into this at a later time. Not up to it this week. I recently fractured a rib... so not able to spend alot of time at my desk. Been mostly on the laptop in bed.